shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.shardingsphere.proxy.backend.text.distsql.ral.common.updatable;
+
+
+import org.apache.shardingsphere.distsql.parser.statement.ral.common.updatable.AlterInstanceStatement;
+import org.apache.shardingsphere.mode.manager.ContextManager;
+import org.apache.shardingsphere.mode.metadata.persist.MetaDataPersistService;
+import org.apache.shardingsphere.mode.persist.PersistRepository;
+import org.apache.shardingsphere.proxy.backend.context.ProxyContext;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.*;
+
+import static org.mockito.Mockito.*;
+
+public class AlterInstanceHandlerTest {
```

Comment:
@ismailalammar Please add final for AlterInstanceHandlerTest.

Reply:
resolved